src/main/java/gui/modal/StartupModalView.java

```diff
@@ -0,0 +1,180 @@
+package gui.modal;
+
+import gui.root.RootPane;
+import gui.styling.StyledButton;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.control.Label;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.VBox;
+import javafx.scene.paint.Color;
+import lombok.Getter;
+
+public class StartupModalView extends ModalView {
+
+    /*
+     * Tweakable styling variables.
```

Comment:
Big and noticeable. I prefer it this way for all touchable/non-touchable variables.